src/item_tname.cpp

```diff
@@ -119,6 +119,26 @@ std::string burn( item const &it, unsigned int /* quantity */,
     return {};
 }
 
+std::string custom_item_prefix( item const &it, unsigned int /* quantity */,
+                                segment_bitset const &/* segments */ )
+{
+    std::string prefix;
+    for( const flag_id &f : it.get_prefix_flags() ) {
+        prefix += f->item_prefix().translated();
```

Comment:
You probably want some whitespace between labels

Reply:
I'm considering that sometimes Modders might not want whitespaces. Let me go and ask others for their opinions...

Reply:
<img width="185" alt="image" src="https://github.com/CleverRaven/Cataclysm-DDA/assets/81861009/c031c160-cb36-44ae-9acb-84533575187a">

When use it to change the color of item name, there's no need to add whitespace.
eg.
prefix: <color_red>
suffix: </color>

Reply:
Not all languages use space to delimit words and as @PipeYume said some affixes do not require a space, so any space should be specified in JSON.